CONTRIBUTING.md

```diff
@@ -1,6 +1,89 @@
 # How to contribute
 
-## Prerequisites
+Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved!
+
+## Using the issue tracker
+
+Use the issues tracker for:
+
+* [bug reports](#bug-reports)
+* [feature requests](#feature-requests)
+* [submitting pull requests](#pull-requests)
+
+Personal support request should be discussed on [F# Software Foundation Slack](https://fsharp.org/guides/slack/).
+
+## Bug reports
+
+A bug is either a _demonstrable problem_ that is caused in Ionide failing to provide the expected feature or indicate missing, unclear, or misleading documentation. Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the `master` branch in the repository.
+
+3. **Isolate and report the problem** &mdash; ideally create a reduced test case.
+
+Please try to be as detailed as possible in your report. Include information about
+your Operating System, as well as your `dotnet` (or `mono` \ .Net Framework), and F# versions. Please provide steps to
+reproduce the issue as well as the outcome you were expecting! All these details
+will help developers to fix any potential bugs.
+
+Ionide provide an easy way to gather all this informations:
+
+Do `Ctrl+Shift+P > F#: Get info for diagnostics`, this will open a file with something like:
+
+> <!-- Please copy/paste this file content into a Github issue -->
+> ### Problem
+>
+> <!-- Describe here your problem -->
+>
+> ### Steps to reproduce
+>
+> <!-- Add here the step to reproduce you problem. Example: -->
+> <!-- 1. Open an F# file -->
+> <!-- 2. Ctrl + P > "F# Add Reference" -->
+>
+> ### Machine infos
+>
+> * Operating system: **Darwin**
+> * Arch: **x64**
+> * VSCode: **1.23.1**
+> * Runtime: **netcore**
+> * Dotnet version: **2.1.103
+> **
```

Comment:
This shouldn't be on new line, I guess

Reply:
Ah yes, this is a bug in the Diagnostic generator. 

I will fix it and also the name of the file open is incorrect. ATM it's: Diagnostric info

Will update the PR later